installer/packer/scripts/kov/configure_kov.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/bash
```

Comment:
We do indeed support the VI admin to change OVF params after deployment (like replacing ssl certs or changing port number of a service).